hangman.py

```diff
@@ -48,6 +48,8 @@ def word_is_solved(word, guesses):
     """
     pass
```

Comment:
Need to remove `pass`.

Reply:
`pass` is a placeholder than needs to be removed. Remove it an the code is good.